comptests/motr/test_motr_k8s_sanity.py

```diff
@@ -257,3 +298,22 @@ def test_cluster_shutdown_with_m0crate(self):
         for node in self.motr_obj.get_node_pod_dict():
             self.update_m0crate_config(config_file, node)
             self.motr_obj.m0crate_run(config_file, remote_file, node)
+
```

Comment:
Please add lc marker

Reply:
we don't want to group it with other markers as it will fail motr tests tests since cluster will not be deployed with motr_clients in other cases.